fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -26,6 +29,13 @@
 public final class FHIRRegistry {
     private static final Logger log = Logger.getLogger(FHIRRegistry.class.getName());
 
+    private static final Comparator<Canonical> CANONICAL_COMPARATOR = new Comparator<Canonical>() {
+        @Override
+        public int compare(Canonical first, Canonical second) {
+            return first.getValue().compareTo(second.getValue());
```

Comment:
should it verify that the urls match before it compares them?
should it try to parse out the version part of the canonicals and compare those?

Reply:
should it try to parse out the version part of the canonicals and compare those?

Reply:
for the purpose of sorting (which is all this is used for), the value of the canonical can be compared.